### PR TITLE
Only run compat sandbox(classic)? on postgres tests on Linx

### PR DESCRIPTION
**Pull request by cocreature**
They keep causing timeouts on MacOS and nobody seems to consider them
useful enough to investigate. For now I only disabled the postgres
tests because I haven’t seen the others time out but if they do I’ll
add them as well

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by stefanobaghino-da — APPROVED** [review]

**Review by garyverhaegen-da — APPROVED** [review]
I'm always for disabling tests, but shouldn't we get approval from the relevant team before merging?

**Comment by stefanobaghino-da** [comment]
> I'm always for disabling tests, but shouldn't we get approval from the relevant team before merging?

Approval came off band. /cc @nmarton-da

**Comment by garyverhaegen-da** [comment]
> > I'm always for disabling tests, but shouldn't we get approval from the relevant team before merging?
> 
> Approval came off band. /cc @nmarton-da

That band dies every 15 days so I'd prefer having an approval here, if possible.

**Comment by nmarton-da** [comment]
Since it is merged, pls have this message as my approval. :ok_hand:

**Pull request merged**
